docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -237,3 +237,15 @@ curl -XGET 'localhost:9200/index/type/_search'
 }
 ---------------
 
+=== Codecs
+
+It is not possible to specify per-field postings and doc values formats in the
```

Comment:
Say "no longer possible" instead of "not possible"? You might also be more explicit about what "old indices" means. Also, typo: tail -> fail